.github/workflows/workspace.yml

```diff
@@ -48,7 +48,23 @@ jobs:
     - name: Web WASM (wasm_js.rs)
       env:
         RUSTFLAGS: -Dwarnings --cfg getrandom_backend="wasm_js"
-      run: cargo clippy -Zbuild-std --target wasm32-unknown-unknown
+      run: cargo clippy --features std -Zbuild-std=panic_abort,std --target wasm32-unknown-unknown
+    - name: Web WASM no_std (wasm_js.rs)
+      env:
+        RUSTFLAGS: -Dwarnings --cfg getrandom_backend="wasm_js"
+      run: cargo clippy -Zbuild-std=core,alloc --target wasm32-unknown-unknown
+    - name: Web WASM no_std with atomics (wasm_js.rs)
```

Comment:
These lines do not check any new code with Clippy. If you want to tests that the crate can be built for `wasm32v1-none`, you should do it in the build workflow.